tests/features/frontend_cache_purge.feature

```diff
@@ -515,9 +515,9 @@ Feature:
 
   Scenario: No purge are instructed if a part of the configuration has disappeared.
     Given the following cache purge rules:
-      | Content Type | Paths to Purge         |
-      | page         | /more-basic-pages, /   |
-      | page         | /even-more-basic-pages |
+      | Content Type | Paths to Purge       |
+      | page         | /more-basic-pages, / |
+      | page         |                      |
```

Comment:
Hi @gillesdeudon,
I'm a bit confused with this last change. Other than this, I agree with all the rest